features.ts

```diff
@@ -0,0 +1,34 @@
+import { FeatureSet } from 'lib/feature-flags/feature-flags';
+
+type options = {
+  [key: string]: any;
+};
+
+export const getFeatureFlags = (options?: options): FeatureSet => {
```

Comment:
I'd structure this slightly differently, defining `featureFlags` inside the function so they can take advantage of the options / context object. Something like this:

```tsx
type FeatureFlagContext = {
  environmentName: string;
};

export const getFeatureFlags = ({ environmentName }: FeatureFlagContext): FeatureSet => {
  const featureFlags: FeatureSet = {
    /*
      The feature-flags-implementation-proof has been setup to have an expiry date in the far future.
      The FEATURE-FLAG-EXPIRES comment above will cause ESLint errors once the date in the square brackets has passed.
      Add feature flags below following the format in the example shown.
    */
    // FEATURE-FLAG-EXPIRES [3000-12-31]: feature-flags-implementation-proof
    "feature-flags-implementation-proof": {
      isActive: environmentName === "development",
    },
  };

  return featureFlags;
};
```

Reply:
I see. Thanks! 🙏🏾

Reply:
@chrishutchinson My understanding with this is that the only thing we're passing into `FeatureFlagContext` is the environment name, but this could be a list of keys & values of data to use to set conditions for other feature flags?

Reply:
Yeah that's right, an object of things, but I'm proposing we should be explicit about what those things are (for now)